kubespawner/spawner.py

```diff
@@ -242,19 +237,8 @@ def _namespace_default(self):
                 return f.read().strip()
         return 'default'
 
-    ip = Unicode(
```

Comment:
Why is this removed?

Reply:
It did not look to me that it was being used?  Am I mistaken?